web/src/App.css

```diff
@@ -1,3 +1,9 @@
-.App-header {
+.App {
```

Comment:
We could delete this empty rule but other than that looks awesome!

Reply:
Maybe it's for upcoming styling?